requirements.txt

```diff
@@ -1,4 +1,4 @@
-niworkflows>=0.4.3
+git+https://github.com/poldracklab/niworkflows.git@fdf20103b9dedd4632bb8e26a5eb3b896e349fd2#egg=niworkflows
```

Comment:
0.4.3 has this commit already, please revert this change (I guess the niworkflows dependency has been updated since you sent the PR)